orca-echo/src/test/groovy/com/netflix/spinnaker/orca/echo/spring/EchoNotifyingExecutionListenerSpec.groovy

```diff
@@ -196,4 +196,31 @@ class EchoNotifyingExecutionListenerSpec extends Specification {
     }
     1 * echoService.recordEvent(_)
   }
+
+  def "handles cases with multiple notifications of the same type"() {
```

Comment:
maybe add an extra case where the fields are null, which was the source of the bug

Reply:
They are `null`, because they are unset. But I guess I could have been more explicit about it

Reply:
(Only `address`, `level`, `type` and `when` are set)

Reply:
right 👍